src/api/user_presence.rs

```diff
@@ -13,28 +13,40 @@
 // limitations under the License.
 
 use crate::clock::ClockInt;
-use crate::ctap::Channel;
 use embedded_time::duration::Milliseconds;
 
 pub enum UserPresenceError {
+    // User explicitly declined user presence check.
     Declined,
+    // User presence check was canceled by User Agent.
     Canceled,
+    // User presence check timed out.
```

Comment:
You can use documentation here (`///`) instead of just comments (`//`).

Reply:
Done